src/System.IO.Compression/src/System/IO/Compression/ZipArchive.cs

```diff
@@ -14,22 +14,22 @@ namespace System.IO.Compression
 {
     public class ZipArchive : IDisposable
     {
-        private Stream _archiveStream;
-        private ZipArchiveEntry _archiveStreamOwner;
-        private BinaryReader _archiveReader;
+        private Stream _archiveStream = null!; // Fields initialized in Init() called by constructor
```

Comment:
It looks like Init() is only called from the one ctor?  I'd just manually inline it into the ctor and then revert these changes to fields that are initialized then by the ctor.

Reply:
ok